ILRepack.IntegrationTests/Peverify/PeverifyHelper.cs

```diff
@@ -0,0 +1,49 @@
+﻿using GitHub.Helpers;
+using Microsoft.Win32;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using System.Reactive.Subjects;
+using System.Text.RegularExpressions;
+
+namespace ILRepack.IntegrationTests.Peverify
+{
+    public static class PeverifyHelper
+    {
+        static Regex Success = new Regex(@"All Classes and Methods in .* Verified");
+        static Regex Failure = new Regex(@"\d+ Error\(s\) Verifying .*");
+        public static IObservable<string> Peverify(string workingDirectory, params string[] args)
+        {
+            // TODO better path finding ?
```

Comment:
Use the Registry? 

path: HKLM\Software\Microsoft\Microsoft SDKs\
key: InstallationFolder

http://blogs.msdn.com/b/windowssdk/archive/2008/05/29/windows-sdk-registry-keys.aspx

However, we'll still need to iterate through the *bin*'s subdirectory since the tools directory don't have the same name :(